src/twisted/runner/procmon.py

```diff
@@ -230,9 +234,10 @@ def connectionLost(self, name):
                                                          name)
 
 
-    def startProcess(self, name):
+    def startProcess(self, name, proto = None):
         """
         @param name: The name of the process to be started
+        @param proto: A custom L{twisted.internet.interfaces.ILoggingProcessProtocol} object
```

Comment:
I believe this fixes https://travis-ci.org/twisted/twisted/builds/173581664 - can someone advise me on how to re-run CI, maybe I'm missing a button somewhere?